packages/office-ui-fabric-react/src/components/Button/BaseButton.scss

```diff
@@ -0,0 +1,7 @@
+.flexContainer {
```

Comment:
Is this the root class applied to all Button types? A class like `.baseButton` would be more descriptive if so.